src/components/OrderForm/OrderForm.js

```diff
@@ -1,55 +1,63 @@
-import React, { Component } from 'react';
-
-class OrderForm extends Component {
-  constructor(props) {
-    super();
-    this.props = props;
-    this.state = {
-      name: '',
-      ingredients: []
-    };
-  }
+import { useState } from "react";
 
+function OrderForm() {
+  const [props, setProps] = useState(props);
```

Comment:
Do we need props assigned as part of state here?  I haven't seen that approach before.

Reply:
Thanks for catching that Heather! I'll remove the props from the state.